test/imperative.js

```diff
@@ -489,42 +489,44 @@ metatests.test('dependentSubtests and parallelSubtests are exclusive', test => {
   test.end();
 });
 
-metatests.test('must support dependentSubtests', test => {
-  const t = new metatests.ImperativeTest('mustNotCall test', t => {
-    t.testSync('successful subtest', test.mustCall(t => t.pass()));
-    t.testSync('failing subtest', test.mustCall(t => t.fail()));
-    t.testSync('successful subtest', test.mustNotCall(t => t.pass()));
-  }, { async: false, dependentSubtests: true });
-  t.on('done', () => {
-    test.strictSame(t.success, false);
-    test.end();
+if (!process.browser) {
```

Comment:
Could you put those in a separate files and then just ignore them in metatests-cli config instead of manually checking here.
I think `dependentSubtests.js` and `bailout.js` for names will be fine.